ckan_cloud_operator/deis_ckan/envvars.py

```diff
@@ -57,10 +57,10 @@ def _update(self):
             postgres_host, postgres_port = db_manager.get_internal_proxy_host_port()
             logs.info(f'Connecting to DB proxy: {postgres_host}:{postgres_port}')
         envvars.update(
-            CKAN_SQLALCHEMY_URL=f"postgresql://{db_name}:{db_password}@{postgres_host}:{postgres_port}/{db_name}",
-            CKAN___BEAKER__SESSION__URL=f"postgresql://{db_name}:{db_password}@{postgres_host}:{postgres_port}/{db_name}",
-            CKAN__DATASTORE__READ_URL=f"postgresql://{datastore_ro_user}:{datastore_ro_password}@{postgres_host}:{postgres_port}/{datastore_name}",
-            CKAN__DATASTORE__WRITE_URL=f"postgresql://{datastore_name}:{datastore_password}@{postgres_host}:{postgres_port}/{datastore_name}",
+            CKAN_SQLALCHEMY_URL=f"postgresql://{db_name}@{postgres_host}:{db_password}@{postgres_host}:{postgres_port}/{db_name}",
```

Comment:
Azure SQL instances support usernames only is this format ("username@host"). This format in URI works with patched version of CKAN for Gates.

I'll revert this change anyway, as we currently use helm based CKAN instances instead of deis.